.github/pull_request_template.md

```diff
@@ -1,11 +1,13 @@
 <!--
 
 🌳🌳
```

Comment:
Do you want to change these to 🌲 for Redwood? The change might catch some devs' attention.

Reply:
I looked at this icon but it doesn't look like a redwood tree :)
But for to get some attention I totally agree. Will update.